entrypoint.sh

```diff
@@ -0,0 +1,6 @@
+#!/bin/sh
+
+# See https://stackoverflow.com/a/39398511/740048
+# and https://forum.artixlinux.org/index.php/topic,3360.0.html
+chown -R "$USER_UID":"$USER_GID" "$OSH_HOME"
```

Comment:
I think this goes hand in hand with not allowing the image to execute with a privileged user. See comment on Dockerfile